compose-builder/README.md

```diff
@@ -86,6 +86,8 @@ This folder contains the following environment files:
     This file contains the common environment overrides used by all Edgex services.
 - **common-security.env**
     This file contains the common security related environment overrides used by many Edgex services.
+- **common-sec-bootstrapper.env**
```

Comment:
IMO, we show call this common-sec-stage-gate.env which matches your description

Reply:
ok, I'll rename it and rebase push once beau's PR is in.

Reply:
renamed